lib/msal-core/samples/react-sample-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "react-sample-app",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "msal": "^1.0.1",
```

Comment:
I think this should actually pull from the locally built artifact, instead of the published version.

Reply:
+1, since we use this in development testing as well.

Maybe add a build script that reaches back and out of the sample to make sure the dist file exists?